README.md

```diff
@@ -1,70 +1,104 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Bookstore Application
 
-## Available Scripts
+> "Bookstore Application" is an MVP version of it's kind that allows you to:
+- Display a list of books.
+- Add a book.
+- Remove a selected book.
 
-In the project directory, you can run:
+The application was built using Javascript, React-Redux, SASS.
 
-### `npm start`
+## Project Screenshot
+![screenshot](src/assets/images/screenShot.jpeg)
```

Comment:
- [OPTIONAL] The link to the screenshot is broken, please kindly fix that as this will make your `readme file` look more professional and will portray you as someone who pays attention to details.